spec/integ/crypto/rust-crypto.spec.ts

```diff
@@ -237,6 +247,143 @@ describe("MatrixClient.initRustCrypto", () => {
         // The final call should have progress == total == -1
         expect(progressListener).toHaveBeenLastCalledWith(-1, -1);
     }, 60000);
+
+    describe("Legacy trust migration", () => {
+        beforeEach(() => {
+            fetchMock.reset();
+        });
+
+        it("should not revert to untrusted if legacy was trusted but msk not in cache, big account", async () => {
+            // Just 404 here for the test
```

Comment:
don't quite understand what "for the test" means here (and below, where it has been C&Ped). Obviously this is part of the test... what is the comment telling me?

Reply:
It's just to try with a 404 in one of the tests. BTW what happens if this return a 500? (Done when migrating the cached backup secret) The whole migration fails and something is displayed to the user? like try to refresh?

Reply:
>  The whole migration fails and something is displayed to the user? like try to refresh?

yeah, something like that. It's not pretty.